app/templates/admin/manageServiceLearningFaculty.html

```diff
@@ -111,11 +146,139 @@ <h5 class="ms-3">There are no approved courses for {{ term.description }}.</h5>
+          </td>
+          <td>
+            <select class="form-select" id="{{course.id}}" onchange='changeAction(this)'>
+              <option value="---" disabled selected>Select Action</option>
+              <option value="Modify">Edit</option>
+              <option value="View">View</option>
+              <option value="Print">Print</option>
+          </td>
+        </tr>
+        {% endfor %}
+      </tbody>
+    </table>
+    {% endif %}
+    </div>
+  </div>
+  {% set downloadAndSend = "disabled" if approvedCourses | length == 0 else "" %} 
+
+    <!-- Modal for altering courses -->
+    <div class="modal fade" id="alterModal">
+      <div class="modal-dialog">
+        <div class="modal-content">
+          <div class="modal-header"> Edit Course </div>
+          <form enctype="multipart/form-data" role="form" method="POST">
+            <input type="hidden" id="alterCourseId" name="courseId">
+            <div class="modal-body"> 
+              <table class="table table-striped ml-4">
+                <tbody>
+                  <tr>
+                    <td class="fw-bold">Course Name</td>
+                    <td><input class="form-control" id="courseName" name="courseName"></td>
+                  </tr>
+                  <tr>
+                    <td class="fw-bold"> Course Abbreviation</td>
+                    <td><input class="input form-control" id="courseAbbreviation" disabled></input></td>
+                  </tr>
+                  <tr>
+                    <td class="fw-bold">Service Hours</td>
+                    <td><input class="form-control" id="hoursEarned" name="hoursEarned" type="number" min="0" required/></td>
+                  </tr>
+                  <tr>
+                    <td class="fw-bold">Instructors</td>
+                    <td><input id="courseInstructor" class="form-control" type="search" placeholder="Search instructor" value=""/></td>
+                    <!-- Start of Instructor Table -->
+                    <table class="table mb-3" id="instructorTable" name="instructorTable">
+                      <thead>
+                        <tr>
+                          <th scope="col">Instructor</th>
+                          <th scope="col"></th>
+                        </tr>
+                      </thead>
+                      <tbody id="instructorTableBody">
+                          <tr data-username="" hidden>  <!-- This is a hidden table row. To add new rows to the instructor table, we copy this one and fill out the information -->
+                              <td>
+                                <p class="mb-0"></p>
+                                <input type="text" style="border: none" size="14" class="form-control-sm" id="" name="courseInstructorPhone" aria-label="Instructor Phone" data-value="" value="" placeholder="Phone Number" />
+                                <a class="text-decoration-none primary editButton" tabindex="0" data-username="" id=""  type="button">Edit</a>
```

Comment:
If you're not going to assign a value to id, value, etc., you probably don't need to include it here.